.changeset/selfish-spies-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/backend-defaults': minor
```

Comment:
```suggestion
'@backstage/backend-defaults': patch
```

Let's call this a bugfix, it's clearly not the expected structure and goes against the config schema